tools/unix/generate_localizations.sh

```diff
@@ -66,4 +66,11 @@ cat "$STRINGS_PATH"/{strings,types_strings}.txt> "$MERGED_FILE"
 "$TWINE" generate-all-localization-files --format jquery "$OMIM_PATH/data/countries_names.txt" "$OMIM_PATH/data/countries-strings/"
 "$TWINE" generate-all-localization-files --format jquery "$OMIM_PATH/data/strings/sound.txt" "$OMIM_PATH/data/sound-strings/"
 
+# Generate list of languages and add list in gradle.properties to be used in build.gradle in resConfig
+SUPPORTED_LOCALIZATIONS="supportedLocalizations="$(sed -nE "s/ +([a-zA-Z]{2}(-[a-zA-Z]{2,})?) = .*$/\1,/p" "${OMIM_PATH}/data/strings/strings.txt" | sort -u | tr '\n' ' ' | sed -E 's/, $//' | sed 's/[[:space:]]//g' | tr '-' '_')
```

Comment:
The shorter version is `sed -nEe "s/ +([a-zA-Z]{2}(-[a-zA-Z]{2,})?) = .*$/\1/p" "data/strings/strings.txt" | sort -u | tr '\n' ',' | sed -e 's/-/_/g'`

Reply:
This version add comma after last element :(

Reply:
Just add `-e 's/,$//'` to the last sed invocation.